bootloader/hos/hos.c

```diff
@@ -638,7 +644,8 @@ int hos_launch(ini_sec_t *cfg)
 	_free_launch_components(&ctxt);
 
 	// Copy BCT if debug mode is enabled.
-	memset((void *)0x4003D000, 0, 0x3000);
+	if (ctxt.pkg1_id->kb != KB_FIRMWARE_VERSION_600)
```

Comment:
What happens now on the next update? It memset's again.

Reply:
Isn't that also what the original did? (just without checking fw ver)

Reply:
Doesn't mean its right^^

Reply:
This needs to accommodate and the 6.0.0 version and possible the newer versions.
We don't do this memset for fun on retail units.